backend/utils/sellers.js

```diff
@@ -1,10 +1,11 @@
 const { Seller } = require('../models')
 const { createSalt, hashPassword, checkPassword } = require('../routes/_auth')
 
-async function createSeller({ name, email, password, superuser }) {
+async function createSeller({ name, email, password }, opts) {
   if (!name || !email || !password) {
     return { status: 400, error: 'Invalid registration' }
```

Comment:
Not modified in this PR, but should this be
```
return res.status(400).json({ error: 'Invalid registration' })
```

Same comment on line 14 below.

Reply:
This function is not a route in itself - it's more of a model layer thing. Since it doesn't return to the view, I think it should stay the same as it is now.